media_driver/cmake/linux/media_include_paths_linux.cmake

```diff
@@ -27,6 +27,9 @@ else()
     pkg_check_modules(LIBVA REQUIRED libva>=1.0.0)
     if(LIBVA_FOUND)
         include_directories(BEFORE ${LIBVA_INCLUDE_DIRS})
+        if("${LIBVA_DRIVERS_PATH}" STREQUAL "")
+            pkg_get_variable(LIBVA_DRIVERS_PATH libva driverdir)
```

Comment:
I saw this pkg_get_variable() get the correct LIBVA_DRIVERS_PATH as "/usr/lib/x86_64-linux-gnu/dri" but it cannot pass along to parent cmake
(root CMakefiles.txt uses add_subdirectory(media_driver) so the variables have another scope, though here LIBVA_DRIVERS_PATH get correct value, the counterpart one in up-level is still empty)

Reply:
Thanks, I will fix this.